assets/texts/de/amoDescription.html

```diff
@@ -44,7 +44,7 @@
 <li>Passe den Stil und viele andere Optionen je nach deinen Wünschen an.</li>
 <li>Die Größe des ganzen Emoji-Pickers und der Emojis selbst, ist ebenfalls anpassbar.</li>
 <li>Nutze ein farbiges Toolbar-Icon oder ein dezentes einfarbiges (schwarz/weißes).</li>
-<li>Folgt dem <a href="https://design.firefox.com/photon">Firefox Photon Design</a>.</li>
+<li>Folgt dem <a href="https://acorn.firefox.com/latest/home/acorn-aRSAh0Sp">Firefox Acorn Design</a>.</li>
```

Comment:
Maybe only use https://acorn.firefox.com/ in case that URL changes again, it redirects to the page.